### ISSUE TITLE
Support debugging in Nuclide

### ISSUE THREAD

**Issue by tttpeng** [realm/realm-js#362]
index.js

```
if (typeof Realm != 'undefined') {
    // The global Realm constructor should be available on device (using JavaScriptCore).
    realmConstructor = Realm;  // eslint-disable-line no-undef
} else if (typeof navigator != 'undefined' && navigator.userAgent) { // eslint-disable-line no-undef
    // The userAgent will be defined when running in a browser (such as Chrome debugging mode).
    realmConstructor = require('./browser').default; // (exported as ES6 module)
} else {
    throw new Error('Missing Realm constructor - please ensure RealmReact framework is included!');
}
```

when i use nuclide,I find `navigator.userAgent = undefined`, in nuclide debugging mode,so ...

what can I do?

**Comment by alazier** [comment]
Not sure why this isn't working. Would be nice to be able to support this but for the time being you will need to debug in chrome.

**Comment by wookiem** [comment]
I just hit this one as well.  I was trying to use Nuclide because I am having issues setting breakpoints in Chrome (see link below).  So, unfortunately, the suggested workaround of using Chrome won't help me.

 http://stackoverflow.com/questions/37078774/cant-always-set-react-native-breakpoints-in-chrome

**Comment by toriaezunama** [comment]
Any news on this issue? 
Being able to debug and do everything in one editor is super nice.

**Comment by appden** [comment]
This issue is dependent on #512 being merged and facebook/nuclide#568 being resolved.
